application/backend/src/main/resources/apps/typerefinery/components/widgets/map/README.md

```diff
@@ -156,16 +154,46 @@ These fields are available for input by the authors. These fields are used in te
             <td>This is the URL to edit the flow in Flow Designer.</td>
         </tr>
         <tr>
-            <td>Grid</td>
-            <td>Grid</td>
-            <td></td>
-            <td>Controls grids system.</td>
+            <td rowspan="4"> Grid</td>
+            <td>Width - S breakpoint</td>
+            <td>12 Col</td>
+            <td>Contains small grid width.</td>
         </tr>
         <tr>
-            <td>Style</td>
-            <td>Id</td>
-            <td></td>
-            <td>Controls the css and styling.</td>
+            <td>Width - M breakpoint</td>
+            <td>12 Col</td>
+            <td>Contains medium grid width.</td>
+        </tr>
+        <tr>
+            <td>Width - L breakpoint</td>
+            <td>3 Col</td>
+            <td>Contains large grid width.</td>
+        </tr>
+        <tr>
+            <td>Text Alignment</td>
+            <td>Default</td>
+            <td>Contains alignment of the text.</td>
+        </tr>
+        <tr>
+            <td rowspan="4"> Style</td>
+            <td>Class name</td>
+            <td>None</td>
+            <td>Add custom style classes.</td>
+        </tr>
```

Comment:
![image](https://user-images.githubusercontent.com/54505967/228525874-a272b49e-065d-4615-a6d4-911b95bc3e65.png)

Why map component is updated with these styles ?

Reply:
now its changed.